Allow to disable rules withing the scope of the current block (#887)
MarketSquare/robotframework-robocop: docs/including_rules.rst

Ignored blocks can partly overlap. Rule name and rule id can be used interchangeably.

> It is possible to ignore whole file if you start file with ``# robocop: disable``.

Author: Added sentence along with example (I have tried to make it more real world like to adhere to #888)

Ignored blocks can partly overlap. Rule name and rule id can be used interchangeably.

The disabler is aware of the context where it was called, and it will be enabled again at the end of the current code

> block (such as keyword, test case, "for" and "while" loop and "if" statement). In following example,
> ``wrong-case-in-keyword-name`` disabler will be enabled only inside "for" loop.

Reviewer: ```suggestion
block (such as keyword, test case, "for" and "while" loops and "if" statement). In the following example,
``wrong-case-in-keyword-name`` disabler will disable the rule only inside the "for" loop.
```
